src/REPLMode/REPLMode.jl

```diff
@@ -453,7 +454,7 @@ struct MiniREPL <: REPL.AbstractREPL
     t::REPL.Terminals.TTYTerminal
 end
 function MiniREPL()
-    MiniREPL(TextDisplay(stdout), REPL.Terminals.TTYTerminal(get(ENV, "TERM", Sys.iswindows() ? "" : "dumb"), stdin, stdout, stderr))
+    MiniREPL(TextDisplay(stdout_f()), REPL.Terminals.TTYTerminal(get(ENV, "TERM", Sys.iswindows() ? "" : "dumb"), stdin, stdout_f(), stderr_f()))
 end
```

Comment:
I don't know why the `pkg""` tests are still printing after changing this ^ https://github.com/JuliaLang/Pkg.jl/blob/0dc6150bd56fa89c37a0b86da8036d68df1bdcea/test/repl.jl#L17

Reply:
I guess one has to investigate 

https://github.com/JuliaLang/Pkg.jl/blob/0dc6150bd56fa89c37a0b86da8036d68df1bdcea/src/REPLMode/REPLMode.jl#L421

and

https://github.com/JuliaLang/Pkg.jl/blob/0dc6150bd56fa89c37a0b86da8036d68df1bdcea/src/REPLMode/REPLMode.jl#L441

and figure out where that gets printed in the end.

Reply:
Looks like `Pkg.REPLMode.minirepl[]` just needed resetting because the default io changed after it was intiially set in `REPLMode.__init__()`